CGPs/cgp-0044.md

```diff
@@ -11,38 +11,46 @@ date-executed: [if executed] <date created on, in ISO 8601 (yyyy-mm-dd) format>
 
 ## Overview
 
-This Governance intends to increase the amount of the buckets, as a percent of the total amount of the unfrozen Reserve Celo buckets. The concrete proposal is to lift the current bucket sizes to `X%` of the total unfrozen CELO in the Reserve.
+This Governance proposal aims to reduce the slippage incurred when trading Celo stable assets, that is cUSD and cEUR, via the Celo stability mechanism (often referred to as Mento). All else equal, a reduction of the Mento slippage, which can be achieved via an increase of the Mento `reserveFraction` parameter, can be expected to tighten the arbitrage bounds on the cUSD/USD and cEUR/EUR pairs (collectively referred to as cXXX/XXX in this proposal).
```

Comment:
`cXXX/XXX` is no longer holds with cREAL :(

Reply:
Uhhh true. But should be ok for this proposal right?